kotest-framework/kotest-framework-engine/src/jvmMain/kotlin/io/kotest/engine/config/applyConfigFromAutoScan.kt

```diff
@@ -17,12 +17,17 @@ import io.kotest.mpp.log
 internal actual fun applyConfigFromAutoScan(configuration: ProjectConfiguration) {
 
    // this property is used to disable class path scanning for configurations
-   if (System.getProperty(KotestEngineProperties.disableAutoScanClassPathScanning) == "true") {
+   if (System.getProperty(KotestEngineProperties.disableAutoScanClassPathScanning) == true.toString()) {
```

Comment:
What's the reason for this?

Reply:
This is unnecessary indeed but it avoids doing the same mistake as the author did with the property name that is using string literals.

Let me know and I’ll revert this line.

I’d expect even `True` or `TRUE` to resolve to true and this is the first step towards that. (But I didn’t want to introduce a breaking change. So I ended up doing just a half step.)